libs/fabric/src/lib/components/dropdown/directives/dropdown-options.directive.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { ContentChildren, Directive, QueryList } from '@angular/core';
+
+import { DropdownOptionDirective } from './dropdown-option.directive';
+import { IDropdownOption } from 'office-ui-fabric-react';
+
+/**
+ * Wrapper directive for creating multiple DropdownOptions
+ */
+@Directive({ selector: 'fab-dropdown > options' })
+export class DropdownOptionsDirective {
+  @ContentChildren(DropdownOptionDirective) readonly directiveItems: QueryList<DropdownOptionDirective>;
+
+  get items() {
+    return this.directiveItems.map<IDropdownOption>(directiveItem => {
+      return {
+        key: directiveItem.optionKey,
+        text: directiveItem.text,
```

Comment:
WDYT about replacing these with spreading (`...`) the `directiveItem`? Except for the `key` -> `optionKey` of course.

Reply:
Style preference on my end. I don't like using the … unless it's exhaustive for all items.

Reply:
You could do the following:
```typescript
return this.directiveItems.map<IDropdownOption>(({ optionKey, ...otherDirectiveProps }) => ({
  key: optionKey,
  ...otherDirectiveProps
}))
```

With simple object destructuring you can get omitting props pretty easily, and then you have all props in `otherDirectiveProps`, except for the ones that you explicitly need to rename.

WDYT?

Reply:
This seems fine to my sensibilities, and I agree with your logic. I'll make the change.